cli/flags.go

```diff
@@ -233,9 +240,17 @@ type Var[T any] struct {
 	EnvVar  string
 	Target  *T
 
+	// Parser and Printer are the generic functions for converting string values
+	// to/from the target value. These are populated by the individual flag
+	// helpers.
 	Parser  ParserFunc[T]
 	Printer PrinterFunc[T]
 
+	// Predict is the completion predictor. If no predictor is defined, it
+	// defaults to predicting "something" for all flags except boolean flags.
```

Comment:
Is "something" a technical term of art here? It's not immediately obvious to me what this means.

EDIT: from the below code, I think it means predict.Something, perhaps that could be more clear.

Reply:
Done PTAL